index/arbitrum-nova/erc1155.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
lets keep these files please. they are empty, but important we keep them as stubs